test/test_ops.py

```diff
@@ -1763,22 +1861,84 @@ class TestRefsOpsInfo(TestCase):
     @parametrize("op", ref_ops_names)
     def test_refs_are_in_python_ref_db(self, op):
         if op in self.skip_ref_ops:
-            raise unittest.SkipTest(f"{op} does not have an entry in python_ref_db")
-        self.assertIn(op, self.ref_db_names)
-
-    @parametrize("op", ref_ops_names)
-    def test_refs_are_in_decomp_table(self, op):
-        path = op.split('.')
-        module_path = '.'.join(path[:-1])
-        op_name = path[-1]
-        op_impl = getattr(import_module(f"torch.{module_path}"), op_name)
-
-        if op in self.not_in_decomp_table:
-            self.assertNotIn(op_impl, torch._decomp.decomposition_table.values(),
-                             f"Unexpectedly found {op} in torch._decomp.decomposition_table.values()")
-        else:
-            self.assertIn(op_impl, torch._decomp.decomposition_table.values(),
-                          f"Did not find {op} in torch._decomp.decomposition_table.values()")
+            raise unittest.SkipTest(f"{op} skipped")
+        # Skip aliases since we don't want them to have dedicated RefInfo
+        # entries.  Instead, refs are expected to follow the same format as
+        # OpInfos, where each ref alias is part of its corresponding ref entry.
+        # If we had separate entries just for aliases, it would result in too
+        # much boilerplate for no good reason, e.g., due to handling skips.
+        if op in self.ref_alias_names:
+            return
+        self.assertIn(op, self.ref_db_names,
+                      msg=f"{op} does not have an entry in python_ref_db")
+
+    # like 'test_refs_are_in_python_ref_db' but check in the other direction,
+    # this includes aliases since those are not allowed in 'ref_db_names'
+    @parametrize("op", itertools.chain(ref_db_names, ref_alias_names))
+    def test_python_refs_are_in_ops_names(self, op):
+        # TODO: skip nvfuser refs since those don't have __all__ and it's not
+        # clear at the moment what would be the best way to check those
+        # https://github.com/pytorch/pytorch/issues/85081
+        if op.startswith("ops.nvprims"):
+            raise unittest.SkipTest(f"Skipping nvfuser ref: {op}")
+        self.assertIn(op, self.ref_ops_names)
```

Comment:
I think this test may be better not to be parametrised, and simply have it execute once and return a list of ops that are problematic, together with a nice message explaining why did it fail and how to fix it, similar to the test following it.